model/version.go

```diff
@@ -13,6 +13,8 @@ import (
 // It should be maintained in chronological order with most current
 // release at the front of the list.
 var versions = []string{
+	"5.18.0",
```

Comment:
why adding this? it is not clear to me

Reply:
Mostly plugin. Plugins use [`model.CurrentVersion`](https://github.com/mattermost/mattermost-server/blob/0dcff49c01de56e9379e1fc248847e26f5c71c42/model/version.go#L75) to check if they are compatible with the running server. Because changes that are going into `master` right now will be in 5.18 (probably), we want to bump this version to allow plugin to check, if they are running against a 5.18 server.